plugins/by-name/sqlite-lua/default.nix

```diff
@@ -0,0 +1,21 @@
+{
+  lib,
+  pkgs,
+  ...
+}:
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "sqlite-lua";
+  originalName = "sqlite.lua";
+  luaName = "sqlite.lua";
+  package = lib.mkPackageOption pkgs "sqlite-lua" {
+    default = [
+      "vimPlugins"
+      "sqlite-lua"
+    ];
+  };
```

Comment:
```suggestion
  package = "sqlite-lua";
```

Reply:
Oh yeah, the main package option is a little smarter, isn't it?